README.md

```diff
@@ -1,157 +1,179 @@
+| [ActiveRecord](#active-record)  | activerecord-opentracing               | ~> 5.0             |
+| [Elasticsearch](#elasticsearch) | signalfx-elasticsearch-instrumentation | >= 6.0.2           |
+| [Faraday](#faraday)             | signalfx-faraday-instrumentation       | >= 0.9.0           |
+| [Grape](#grape)                 | grape-instrumentation                  | >= 0.13.0          |
+| [Mongo](#mongo)                 | mongodb-instrumentation                | >= 2.1.0           |
+| [Mysql2](#mysql2)               | mysql2-instrumentation                 | >= 0.4.0           |
+| [Net::HTTP](#nethttp)           | nethttp-instrumentation                | Ruby >= 2.0        |
+| [Pg](#pg)                       | pg-instrumentation                     | >= 0.18.0          |
+| [Rack](#rack)                   | rack-tracer                            | >= 0.1             |
+| [Rails](#rails)                 | rails-instrumentation                  | >= 3.0.0           |
+| [Redis](#redis)                 | redis-instrumentation                  | >= 4.0.0           |
+| [RestClient](#restclient)       | restclient-instrumentation             | >= 1.5.0           |
+| [Sequel](#sequel)               | sequel-instrumentation                 | >= 3.47.0          |
+| [Sidekiq](#sidekiq)             | sfx-sidekiq-opentracing                | >= 0.7.0           |
+| [Sinatra](#sinatra)             | sinatra-instrumentation                | >= 1.0.0           |
+
+## Configuration values
+
+| Configure parameter | Environment variable  | Default                          | Notes |
+| ------------------- | --------------------  | -------------------------------- | ----- |
+| tracer              | N/A                   | `nil`                            | The OpenTracing global tracer. |
+| ingest_url          | SIGNALFX_ENDPOINT_URL | `http://localhost:9080/v1/trace` | The endpoint the tracer sends spans to. Send spans to a Smart Agent, OpenTelemetry Collector, or a SignalFx ingest endpoint. |
+| service_name        | SIGNALFX_SERVICE_NAME | `signalfx-ruby-tracing`          | The name to identify the service in SignalFx. |
+| access_token        | SIGNALFX_ACCESS_TOKEN | `''`                             | The SignalFx organization access token. |
+
+## Install the SignalFx Tracing Library for Ruby
+
+Follow these steps to install the tracing library. You can either use the
+bootstrap utility to install the tracing library and its dependencies or
+manually install everything. 
+
+The bootstrap utility updates your Gemfile and installs the required
+dependencies for you. For information about the bootstrap utility, see the
+[sfx-rb-trace-bootstrap](bin/sfx-rb-trace-bootstrap) file.
+
+The steps assume you have RubyGems and Bundler.
+
+### Install the library with the bootstrap utility
+
+1. Set the service name to identify the service in SignalFx:
+   ```bash
+   $ export SIGNALFX_SERVICE_NAME = "yourService"
```

Comment:
As with all other libraries setting the environment variables is optional an only relevant to running the actual ruby app.  It doesn't constitute the first step, which is installing the library and instrumentations.

Reply:
The endpoint URL and access token don't seem optional.

Reply:
If deviating from the defaults/sending to SFx directly and not modifying a configure call yes, but again that's only for the actual running application and not during installation.